src/GitLink/Pdb/SrcSrv.cs

```diff
@@ -1,25 +1,24 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
 // <copyright file="SrcSrv.cs" company="CatenaLogic">
-//   Copyright (c) 2014 - 2014 CatenaLogic. All rights reserved.
+//   Copyright (c) 2014 - 2016 CatenaLogic. All rights reserved.
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
-
 namespace GitLink.Pdb
 {
     using System;
     using System.Collections.Generic;
     using System.IO;
     using Catel;
 
-    public static class SrcSrv
+    internal static class SrcSrv
     {
         private static string CreateTarget(string rawUrl, string revision)
         {
             return string.Format(rawUrl, revision);
         }
 
-        public static byte[] Create(string rawUrl, string revision, IEnumerable<Tuple<string, string>> paths, bool downloadWithPowershell)
+        internal static byte[] Create(string rawUrl, string revision, IEnumerable<Tuple<string, string>> paths, bool downloadWithPowershell)
```

Comment:
I thought we had a SrcSrvContext for this? The reason we introduced it is that we don't create breaking changes if we need an additional parameter. On the other side, it's internal now.